core/vibes/soul/sections/not-found/index.tsx

```diff
@@ -25,15 +29,22 @@ export function NotFound({
   subtitle = "Take a look around if you're lost.",
   className = '',
 }: NotFoundProps) {
+  const { setIsSearchOpen } = useSearch();
+
+  const handleOpenSearch = () => {
+    setIsSearchOpen(true);
+  };
+
   return (
     <SectionLayout className={className} containerSize="2xl">
       <header className="font-[family-name:var(--not-found-font-family,var(--font-family-body))]">
         <h1 className="mb-3 font-[family-name:var(--not-found-title-font-family,var(--font-family-heading))] text-3xl leading-none font-medium text-[var(--not-found-title,hsl(var(--foreground)))] @xl:text-4xl @4xl:text-5xl">
           {title}
         </h1>
-        <p className="text-lg text-[var(--not-found-subtitle,hsl(var(--contrast-500)))]">
+        <p className="mb-4 text-lg text-[var(--not-found-subtitle,hsl(var(--contrast-500)))]">
           {subtitle}
         </p>
+        <Button onClick={handleOpenSearch}>Search</Button>
```

Comment:
Should we change the behavior to toggle?

Reply:
Sorry, what do you mean?

Reply:
Right now when you click the search icon in the header, it toggles the popover open and closed:

https://github.com/user-attachments/assets/88f2475b-f675-4a09-9a67-36d2f08250ef

But the search button on the 404 page just sets open to `true`, no toggle:

https://github.com/user-attachments/assets/aad9df65-efd0-423a-b34f-4817f72a3ead

Reply:
Going to merge as-is for now, we can always modify later 👍